crates/sui-storage/src/indexes.rs

```diff
@@ -45,6 +51,14 @@ pub struct ObjectIndexChanges {
     pub new_dynamic_fields: Vec<(DynamicFieldKey, DynamicFieldInfo)>,
 }
 
+#[derive(Clone, Serialize, Deserialize, Ord, PartialOrd, Eq, PartialEq, Debug)]
+pub struct CoinInfo {
```

Comment:
Should we make this an enum, so we can evolve this struct later with more fields if needed